API_MO/conventions/SimpleFreeFieldHRSOS_1.0.csv

```diff
@@ -1,43 +1,44 @@
 GLOBAL:DataType	SOS	rm	  	attribute	Filters described as second-order section (SOS) coefficients
-GLOBAL:History			 	attribute	
-GLOBAL:License	No license provided, ask the author for permission	m	  	attribute	
-GLOBAL:Organization		m	  	attribute	
-GLOBAL:References				attribute	
-GLOBAL:RoomType	free field	m	  	attribute	
-GLOBAL:Origin				attribute	
-GLOBAL:DateCreated		m	 	attribute	
-GLOBAL:DateModified		m	 	attribute	
-GLOBAL:Title		m		attribute	
-ListenerPosition	[0 0 0] 	m	IC, MC	double	
-ListenerPosition:Type	cartesian	m	  	attribute	
-ListenerPosition:Units	metre	m	  	attribute	
-ReceiverPosition	[0 0.09 0; 0 -0.09 0]	m	rCI, rCM	double	
-ReceiverPosition:Type	cartesian	m	  	attribute	
-ReceiverPosition:Units	metre	m	  	attribute	
+GLOBAL:History			 	attribute
+GLOBAL:License	No license provided, ask the author for permission	m	  	attribute
+GLOBAL:Organization		m	  	attribute
+GLOBAL:References				attribute
+GLOBAL:RoomType	free field	m	  	attribute
+GLOBAL:Origin				attribute
+GLOBAL:DateCreated		m	 	attribute
+GLOBAL:DateModified		m	 	attribute
+GLOBAL:Title		m		attribute
+ListenerPosition	[0 0 0] 	m	IC, MC	double
+ListenerPosition:Type	cartesian	m	  	attribute
+ListenerPosition:Units	metre	m	  	attribute
+ReceiverPosition	[0 0.09 0; 0 -0.09 0]	m	rCI, rCM	double
+ReceiverPosition:Type	cartesian	m	  	attribute
+ReceiverPosition:Units	metre	m	  	attribute
 SourcePosition	[0 0 1]	m	IC, MC	double	Source position is assumed to vary for different directions/positions around the listener
-SourcePosition:Type	spherical	m	  	attribute	
-SourcePosition:Units	degree, degree, metre	m	  	attribute	
-EmitterPosition	[0 0 0]	m	eCI, eCM	double	
-EmitterPosition:Type	cartesian	m	  	attribute	
-EmitterPosition:Units	metre	m	  	attribute	
+SourcePosition:Type	spherical	m	  	attribute
+SourcePosition:Units	degree, degree, metre	m	  	attribute
+EmitterPosition	[0 0 0]	m	eCI, eCM	double
+EmitterPosition:Type	cartesian	m	  	attribute
+EmitterPosition:Units	metre	m	  	attribute
 GLOBAL:DatabaseName		m	  	attribute	name of the database to which these data belong
 GLOBAL:ListenerShortName		m	  	attribute	ID of the subject from the database
-ListenerUp	[0 0 1]	m	IC, MC	double	
-ListenerView	[1 0 0]	m	IC, MC	double	
-ListenerView:Type	cartesian	m		attribute	
-ListenerView:Units	metre	m		attribute	
+ListenerUp	[0 0 1]	m	IC, MC	double
+ListenerView	[1 0 0]	m	IC, MC	double
+ListenerView:Type	cartesian	m		attribute
+ListenerView:Units	metre	m		attribute
 Data.SOS	permute([0 0 0 1 0 0; 0 0 0 1 0 0], [3 1 2]);	m	mRn	double	Filter coefficients as SOS coefficients.
 Data.SamplingRate	48000	m	I, M	double	Sampling rate of the coefficients in Data.SOS and the delay in Data.Delay
-Data.SamplingRate:Units	hertz	m		attribute	
+Data.SamplingRate:Units	hertz	m		attribute
 Data.Delay	[0 0]	m	IR, MR	double	Broadband delay (in samples resulting from SamplingRate)
+N	6	m	I	double	Total number of SOS coefficients. Shall be greater or equal six and in integer multiple of 6
```

Comment:
This is a mistake in AES69-2020. The SOS conventions should not have the Variable N since it is a redundant information.

```suggestion
```

Reply:
This will be resolved after the next standards Meeting